agent/local/state_test.go

```diff
@@ -1912,6 +1912,59 @@ func TestAgent_AliasCheck(t *testing.T) {
 	}
 }
 
+func TestAgent_AliasCheck_ServiceNotification(t *testing.T) {
```

Comment:
if you comment out the patch, does this test fail?

Reply:
Yep this would fail. We only used to notify alias checks if checks were updated. Now we notify about changes to aliased services as well.